### DISCUSSION TITLE
You know your life is over... - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#76] [General]
# You know your life is over... - Alan Pope's blog

Every so often my brain reminds me of a conversation from long ago. Sometimes I’ll go for months without thinking about it, but then it’ll trigger, and I can’t stop thinking about it. It happened this week.
Many years ago I taught technical courses for SAP in their London training centre. There’d often be moments during the day when the students were busy doing exercises and off-topic conversations would start. Here’s how one went down.

https://popey.com/blog/2023/09/you-know-your-life-is-over/

**Comment by Hund** [comment]
You know that your life is over when you watch football on a Saturday.

It's (for most people) only one of TWO free days per week. And they waste it by watching other people being healthy.. 

Jokes aside. It's your free time, you should be able to do whatever you want with it, no matter what it is.
